blocks/library/block/index.js

```diff
@@ -15,7 +14,8 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { getBlockType, registerBlockType, hasBlockSupport, getBlockDefaultClassname } from '../../api';
+import BlockEdit from '../../block-edit';
```

Comment:
👍 

Is this exposed? How do we use this component when moved to the `editor` module :)

Reply:
Yes, it is exposed: https://github.com/WordPress/gutenberg/blob/master/blocks/index.js#L21.